client/src/chat.ts

```diff
@@ -12,7 +12,7 @@ import globals from './globals';
 import * as modals from './modals';
 
 // Variables
-const emojiMap = new Map();
+const emojiMap = new Map<string, string>();
```

Comment:
This is a bit verbose, but it makes it blatantly clear what's the expected type. In this case, emojiMap was <any,any>

Reply:
nice work finding this!